int32 button	#0=Head Front;2=Head Middle;3=Head Back
bool state	#True pressed; False released   